eZ/Publish/Core/Search/Elasticsearch/Content/Extractor/Loading.php

```diff
@@ -17,7 +17,7 @@
 
 /**
  * The Loading Extractor extracts the value object from the Elasticsearch search hit data
- * by loading it from the database.
+ * by loading it from the persistence.
```

Comment:
Fixed in https://github.com/ezsystems/ezplatform-solr-search-engine/commit/2a1b7822ea81e017b7be58352dc00075326c65ae